tokens/sys/space.json

```diff
@@ -2,55 +2,68 @@
   "space": {
     "zero": {
       "value": "0",
-      "type": "spacing"
+      "type": "spacing",
```

Comment:
Question: I think we changed all our language to be `space` instead of `spacing`. Is this what we want?

Reply:
This is a tokens studio thing – disregard.